django_cyverse_auth/authBackends.py

```diff
@@ -473,3 +475,13 @@ def _update_token(self, auth_url, username, token, expiry_time, request=None):
     def _grant_access(self, auth_url, username):
         user_profile = self._user_profile_for_auth(auth_url, username)
         return get_or_create_user(username, user_profile)
+
+    def _lookup_email(self, session):
+        try:
+            user_id = str(session.get_user_id())
+            ks_client = client.Client(session=session)
+            ks_user = ks_client.users.get(user_id)
+            email = str(ks_user.email)
+        except:
+            raise Exception("Cannot get user's email")
```

Comment:
If the goal is to `raise Exception` on failure, then it is probably best to _remove_ this try/except and have the application using this library handle:
```
try:
  keystone_validate_...
  do_success_logic()
except keystoneclient.Exception
  error_info = get_error_specific_info()
  do_fail_logic(error_info)
```